test/form_builder/general_test.rb

```diff
@@ -250,6 +251,7 @@ def with_custom_form_for(object, *args, &block)
   test 'builder generates file for attributes that are real db columns but have file methods' do
     @user.home_picture = MiniTest::Mock.new
     @user.home_picture.expect(:mounted_as, true)
+    @user.home_picture.expect(:!, false)
```

Comment:
I'm confused with this. What is this expecting?

Reply:
Without this the tests are failing:
<details>
<summary>test_builder_generates_file_for_attributes_that_are_real_db_columns_but_have_file_methods</summary>

```sh
FormBuilderTest#test_builder_generates_file_for_attributes_that_are_real_db_columns_but_have_file_methods:
NoMethodError: unmocked method :!, expected one of [:mounted_as]
    /mac/.rvm/gems/ruby-2.4.3@simple_form/gems/activesupport-5.1.0/lib/active_support/core_ext/object/blank.rb:17:in `blank?'
    /mac/.rvm/gems/ruby-2.4.3@simple_form/gems/activesupport-5.1.0/lib/active_support/core_ext/object/blank.rb:24:in `present?'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/components/errors.rb:18:in `has_value?'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/components/errors.rb:22:in `valid?'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/wrappers/root.rb:33:in `html_classes'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/wrappers/many.rb:59:in `wrap'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/wrappers/many.rb:33:in `render'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/wrappers/root.rb:16:in `render'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/form_builder.rb:123:in `input'
    /mac/code/private/github/m5o__simple_form/test/support/misc_helpers.rb:249:in `block in with_form_for'
    /mac/.rvm/gems/ruby-2.4.3@simple_form/gems/actionview-5.1.0/lib/action_view/helpers/capture_helper.rb:39:in `block in capture'
    /mac/.rvm/gems/ruby-2.4.3@simple_form/gems/actionview-5.1.0/lib/action_view/helpers/capture_helper.rb:203:in `with_output_buffer'
    /mac/.rvm/gems/ruby-2.4.3@simple_form/gems/actionview-5.1.0/lib/action_view/helpers/capture_helper.rb:39:in `capture'
    /mac/.rvm/gems/ruby-2.4.3@simple_form/gems/actionview-5.1.0/lib/action_view/helpers/form_helper.rb:450:in `form_for'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/action_view_extensions/form_helper.rb:27:in `block in simple_form_for'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/action_view_extensions/form_helper.rb:46:in `with_simple_form_field_error_proc'
    /mac/code/private/github/m5o__simple_form/lib/simple_form/action_view_extensions/form_helper.rb:26:in `simple_form_for'
    /mac/code/private/github/m5o__simple_form/test/support/misc_helpers.rb:232:in `with_concat_form_for'
    /mac/code/private/github/m5o__simple_form/test/support/misc_helpers.rb:248:in `with_form_for'
    /mac/code/private/github/m5o__simple_form/test/form_builder/general_test.rb:256:in `block in <class:FormBuilderTest>'
```
</details>